### PR TITLE
Fix golint for pkg/volume/aws_ebs

### PR DESCRIPTION
**Pull request by leakingtapan**
**What this PR does / why we need it**:
Fixes golint issues - pkg/volume/aws_ebs

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Ref #68026

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by leakingtapan** [comment]
updated test/test_owners.csv file as well

**Comment by leakingtapan** [comment]
@justinsb all the tests are passed. Anything else I should do to get this pull request merged?

**Comment by justinsb** [comment]
It needs an OK from an approver in each of the trees you've touched.

I think @lavalamp you are both necessary & sufficient

/assign lavalamp

**Comment by shubheksha** [comment]
@leakingtapan, currently there is a code freeze due to the 1.12 release. Your PR will be merged once it's lifted towards the end of the September.

**Comment by leakingtapan** [comment]
@shubheksha thx for the info. Any links for our code freeze policy? I saw release-1.12 branch is already cut?

**Comment by shubheksha** [comment]
You can find the timeline here: https://github.com/kubernetes/sig-release/blob/master/releases/release-1.12/README.md

File: pkg/volume/awsebs/aws_util.go [hunk 1]
```diff
@@ -45,8 +45,11 @@ const (
 	ebsMaxReplicasInAZ  = 1
 )
 
+// AWSDiskUtil provides operatoins for EBS volume.
 type AWSDiskUtil struct{}
 
+// DeleteVolume deletes an AWS EBS volume.
+// Returns: error
```
Comment (lavalamp) [inline pkg/volume/awsebs/aws_util.go]: nit: we can read the method signature to see it returns an error. The comment should say under what circumstances and what the caller should do about it if it happens.

File: pkg/volume/awsebs/aws_util.go [hunk 2]
```diff
@@ -45,8 +45,11 @@ const (
 	ebsMaxReplicasInAZ  = 1
 )
 
+// AWSDiskUtil provides operatoins for EBS volume.
```
Comment (lavalamp) [inline pkg/volume/awsebs/aws_util.go]: operations

Comment (leakingtapan) [reply pkg/volume/awsebs/aws_util.go]: done

File: pkg/volume/awsebs/aws_util.go (hunk 1 above)
Comment (leakingtapan) [reply pkg/volume/awsebs/aws_util.go]: The circumstances and how caller should handle error is depending on underlying `cloud.DeleteDisk(d.volumeID)`. To not complicate the documentation. I feel its better to remove the line 52.

**Pull request merged**